.test/meta-commands/out.sh

```diff
@@ -137,6 +137,29 @@ jq -s '
 	)
 ' temp/index.json > temp/index.json.new
 mv temp/index.json.new temp/index.json
+SOURCE_DATE_EPOCH=1709081058 \
+	docker buildx build --progress=plain \
+	--provenance=false \
+	--build-context "base:latest=oci-layout://$PWD/temp/@$(jq -r '.manifests[0].digest' index.json)" \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output 'type=oci,tar=false,dest=temp_sbom/' \
```

Comment:
Niiiice, I always forget about `tar=false` -- we should (separately) apply this to the other build so we don't have to extract. :smile:

It's 100% superficial, but I think we should call this directory just `sbom` instead (in hindsight, `temp` was a strange name for our other directory, but we've got a few more external things to port into here like the signing code before we could reasonably swap that).